app/src/pages/common/settingsPage/generalTab/generalTab.scss

```diff
@@ -27,21 +27,21 @@
 
 .button-container {
   box-sizing: content-box;
-  margin-left: 455px;
+  margin-left: 430px;
   &::after {
     content: '';
     clear: both;
     display: table;
   }
   @media (max-width: $SCREEN_SM_MAX) {
-    margin-left: 345px;
+    margin-left: 320px;
   }
   @media (max-width: $SCREEN_XS_MAX) {
     display: none;
   }
 }
 .submit-button {
-  width: 70px;
+  width: 95px;
```

Comment:
I think it will be better to avoid dependency between translated text and the component's static size.
It should be adjusted by content.